### ISSUE TITLE
Auto deploy turtlecoin-api-docs-site on changes to turtlecoin-docs/master

### ISSUE THREAD

**Issue by ar-x** [turtlecoin/turtlecoin-api-docs-site#4]
**tl;dr** Setup a GitHub webhook on `turtlecoin-docs` to point to a AWS lambda function.  Then a commit to `turtlecoin-docs/master` would auto build this project via Travis CI, which would deploy the latest docs.

### Steps

1. Go to https://github.com/turtlecoin/turtlecoin-docs/settings/hooks/new
2. Enter these values:
    * **Payload URL:** https://grj5dp2kbc.execute-api.us-east-1.amazonaws.com/turtlecoin/turtlecoin-wiki-gh-webhook-listener
    * **Content type:** `application/json`
    * **Secret:** some random value. PM me this. This is used by the webhook listener to validate the request from Github request.
    * **Which events would you like to trigger this webhook?:** Just the push event
    * **Active:** checked
3. Press **Add webhook** to save.
4. Let me know.

The code for the webhook listener is [here](https://github.com/turtlecoin/turtlecoin-api-docs-site/blob/turtle-master/lib/wiki_change_responder.js).

**Comment by RocksteadyTC** [comment]
this is done
